flatdata-rs/lib/src/arrayview.rs

```diff
@@ -72,9 +77,10 @@ where
             Bound::Excluded(&idx) => self.data_index(idx + 1),
             Bound::Unbounded => 0,
         };
+        let extra = <T as Struct>::IS_OVERLAPPING_WITH_NEXT as usize;
```

Comment:
1. I think clippy will complain.
2. Is there actually a better name for `extra`. (If not, is fine :))